### PR TITLE
Add ARM64 MSI builds

### LINKED ISSUE #1293: Add ARM64 MSI
Hi there, could you please add MSI file for ARM64 platform into release assets?

### PR DESCRIPTION
**Pull request by breed808**
As WiX v3 did not support ARM64 builds, I've upgraded it to v4 in the Github release action and build script. The bulk of the work was done with `wix convert`.

I've run the release action in my own fork to confirm the MSIs are built correctly, see https://github.com/breed808/windows_exporter/releases/tag/v0.24.1 for the artifacts. I've successfully tested the `amd64` MSI with some variables (`ENABLED_COLLECTORS`, `LISTEN_PORT` and `METRICS_PATH`); unfortunately I'm not in a position to test the `386` or `arm64` MSI packages.

Resolves #1293

### CODE REVIEW

**Review by jkroepke — APPROVED** [review]
LGTM

In a follow up PR, we should think about to drop 386.

**Comment by jkroepke** [comment]
Does it also fix #1320 ?

**Comment by breed808** [comment]
No, but I've fixed that in https://github.com/prometheus-community/windows_exporter/pull/1343/commits/c3e531ec467335331f9eff4f9dab38d814bf689a. Have a re-review and let me know if you're still happy with the changes.

File: installer/windows_exporter.wxs [hunk 1]
```diff
@@ -1,55 +1,42 @@
...
-    <SetProperty Id="ListenFlagBoth" After="InstallFiles" Sequence="execute" Value="--web.listen-address [LISTEN_ADDR]:[LISTEN_PORT]">LISTEN_ADDR AND LISTEN_PORT</SetProperty>
-    <SetProperty Id="ListenFlagAddr" After="InstallFiles" Sequence="execute" Value="--web.listen-address [LISTEN_ADDR]:9182">LISTEN_ADDR AND (NOT LISTEN_PORT)</SetProperty>
-    <SetProperty Id="ListenFlagPort" After="InstallFiles" Sequence="execute" Value="--web.listen-address 0.0.0.0:[LISTEN_PORT]">LISTEN_PORT AND (NOT LISTEN_ADDR)</SetProperty>
+    <Property Id="LISTEN_ADDR" Secure="yes" Value="0.0.0.0" />
+    <Property Id="LISTEN_PORT" Secure="yes" Value="9182" />
+    <SetProperty Id="ListenFlagBoth" After="InstallFiles" Sequence="execute" Value="--web.listen-address [LISTEN_ADDR]:[LISTEN_PORT]" Condition="LISTEN_ADDR AND LISTEN_PORT" />
+    <SetProperty Id="ListenFlagAddr" After="InstallFiles" Sequence="execute" Value="--web.listen-address [LISTEN_ADDR]:9182" Condition="LISTEN_ADDR AND (NOT LISTEN_PORT)" />
+    <SetProperty Id="ListenFlagPort" After="InstallFiles" Sequence="execute" Value="--web.listen-address 0.0.0.0:[LISTEN_PORT]" Condition="LISTEN_PORT AND (NOT LISTEN_ADDR)" />
```
Comment (jkroepke) [inline installer/windows_exporter.wxs:22]: Are the lines required, if the properties have default values now?

**Review by jkroepke — APPROVED** [review]
One question left, LGTM if conversation is resolved.

File: installer/windows_exporter.wxs (hunk 1 above)
Comment (breed808) [reply installer/windows_exporter.wxs:22]: You're correct, these lines are no longer required! I'm amended the most recent commit and successfully re-tested each of the options.

**Pull request merged**